scss/partials/_menu.scss

```diff
@@ -480,17 +484,16 @@
 ------------------------------------------------------------*/
 .nav-overlay {
   animation: sweep-down 0.5s ease-in-out;
-  display: block;
   position: fixed;
   top: 0;
   left: 0;
   width: 100%;
   height: 100%;
   background-color: rgba(0, 0, 0, 0.5);
   z-index: 10;
+  display: none;
 
   @include breakpoint(desktop) {
     top: 10.4rem; // accounts for navbar height
```

Comment:
We need the z-index rule here for the blog, client success, and job filter form since there is a z-index rule applied on the `filter-bar__menu`

Reply:
This is me shooting in the foot. I forgot to reset after playing with the code.